backports/py-vcstool/APKBUILD

```
@@ -0,0 +1,56 @@
+# Contributor: Florent Ferreri <florent@seqsense.com>
+# Maintainer:
+pkgname=py-vcstool
+_pkgname=vcstool
+pkgver=0.3.0
+pkgrel=1
+pkgdesc="Vcstool is a command line tool designed to make working with multiple repositories easier"
+url="https://pypi.org/project/vcstool/"
+arch="noarch"
+license=" Apache-2.0 License"
```

```suggestion
license="Apache-2.0"
```
https://spdx.org/licenses/Apache-2.0.html